deploy/publish.js

```diff
@@ -78,10 +78,11 @@ const main = () => {
   else if (tag === "latest" && dev === true)
     throw new Error(`latest tag can only be used for non-dev versions.`);
 
-  // cp.execSync("pnpm install", {
-  //   cwd: `${__dirname}/../`,
-  //   stdio: "inherit",
-  // });
+  cp.execSync("pnpm install", {
```

Comment:
is it required?

Reply:
We used to run 'pnpm i' individually for each package, but on Monorepo, a single run from root is sufficient, so we made this modification.

Reply:
`pnpm install --filter .` install packages without './package.json'.dependencies

Reply:
We gonna move to the publishing system in Agrntica later so we don't need to modify the script